api/controllers/OpenEAppController.js

```diff
@@ -55,13 +54,19 @@ const OpenEAppController = {
                     daysLeftToDownload
                 );
 
+            const noOfRejectedDocs = OpenEAppController._calculateRejectedDocs(
+                casebookResponse[0]
+            );
+
             res.view('eApostilles/openEApp.ejs', {
                 ...pageData,
                 userRef,
                 user_data: userData,
                 daysLeftToDownload,
                 applicationExpired,
                 applicationStatus: casebookResponse[0].status,
+                allDocumentsRejected:
+                    noOfRejectedDocs == casebookResponse[0].documents.length,
```

Comment:
probably needs a guard condition to protect against casebookResponse[0] = null / undefined OR casebookResponse[0].documents = null / undefined.  There are also similar cases throughout so my first point would be to check instances where you're retrieving casebookResponse[0] and check if this is possible.  Could be a red herring but definitely worth checking to avoid being bitten.